.ko.yaml

```diff
@@ -0,0 +1,9 @@
+builds:
+  - id: conformance-go-client
+    main: ./test/test_images/conformance-go-client
+    env:
+      - CGO_ENABLED=0
```

Comment:
This should be the default with `ko` so I don't think we need to specify it

Reply:
Without it, the tests are failing, at least on my machine.

Reply:
Weird - do you have CGO turn in your environment settings?

https://github.com/google/ko/blob/2f230b88c4891ee3a71b01c1fa65e85e8d6b5f5b/pkg/build/gobuild.go#L381-L386

Reply:
No. I don't:

```
$ echo $CGO_ENABLED
$
```

But, I think it's set by default via go:

```
$ go env CGO_ENABLED
1
```

Same, as well, on `docker.io/library/golang` and `quay.io/fedora/fedora:36` images:

```
$ podman run --rm -it docker.io/library/golang go env CGO_ENABLED
1
```

We are running google/ko from within go code, and there the `CGO_ENABLED` is set to `1` by default.

Reply:
https://github.com/knative-sandbox/reconciler-test/blob/89e41db3a415a3805e420828d695bf14dcf530e5/pkg/images/ko/publish.go#L35-L44

Reply:
> We are running google/ko from within go code, and there the CGO_ENABLED is set to 1 by default.

That's very subtle and I would say our invoker of `ko` remove setting `CGO_ENABLED`